sass/partials/_tools.scss

```diff
@@ -12,6 +12,8 @@
 ///   A CSS `calc()` function that will toggle the proper values
 ///   based on the value of `--ccs-mode--zero` and `--ccs-invert--zero`
 /// @example scss - assign generates theme values that toggle
+///   @use './' as ccs;
```

Comment:
@mirisuzanne I realize this isn't ideal. If we get a new Herman release out that supports `as`, then we can get rid of this.

Reply:
I'll take a look - I bet there's a decent workaround